django-stubs/forms/utils.pyi

```diff
@@ -26,8 +26,16 @@ class RenderableMixin:
         context: dict[str, Any] | None = ...,
         renderer: BaseRenderer | type[BaseRenderer] | None = ...,
     ) -> SafeString: ...
-    __str__ = render
```

Comment:
This is actually a correct thing to do: https://github.com/django/django/blob/e72049aa6302411d8cdf2e798e15fa38e76b92fc/django/forms/utils.py#L57-L58